benchmark_schema.py

```diff
@@ -39,12 +39,12 @@ class TestRun(Base):
     average_memory = Column(Float, nullable=True)
     peak_memory = Column(Float, nullable=True)
 
-    # Timing data (Note: this is not yet collected)
+    # Timing data (not yet collected)
```

Comment:
Hah, this comment is out of date, we are collecting this info now. But I can update separately.